core/api/api.hpp

```diff
@@ -154,6 +161,28 @@ namespace fc::api {
     API_METHOD(SyncSubmitBlock, void, const BlockMsg &)
 
     API_METHOD(WalletSign, Signature, const Address &, const Buffer &)
+
+    /**
+     * Lists the providers in the storage market state
+     * @return vector of StorageProviderInfo
+     */
+    API_METHOD(StateListStorageProviders, std::vector<StorageProviderInfo>)
```

Comment:
```suggestion
    API_METHOD(StateListMiners, std::vector<Address>, const TipsetKey &)
```
Remove `StateListStorageProviders`, call `StateListMiners` and `StateMinerInfo `

Reply:
done